contrib/storage-hive/core/src/main/java/org/apache/drill/exec/store/hive/HiveUtilities.java

```diff
@@ -471,20 +498,28 @@ public static void restoreColumns(HiveTableWithColumnCache table, HivePartition
    * Wrapper around {@link MetaStoreUtils#getSchema(StorageDescriptor, StorageDescriptor, Map, String, String, List)}
    * which also sets columns from table cache to table and returns properties returned by
    * {@link MetaStoreUtils#getSchema(StorageDescriptor, StorageDescriptor, Map, String, String, List)}.
+   *
+   * @param table Hive table with cached columns
+   * @return Hive table metadata
    */
   public static Properties getTableMetadata(HiveTableWithColumnCache table) {
     restoreColumns(table, null);
     return MetaStoreUtils.getSchema(table.getSd(), table.getSd(), table.getParameters(),
       table.getDbName(), table.getTableName(), table.getPartitionKeys());
   }
 
+  /**
+   * Generates unsupported types exception message with list of supported types
+   * and throws user exception.
+   *
+   * @param unsupportedType unsupported type
+   */
   public static void throwUnsupportedHiveDataTypeError(String unsupportedType) {
     StringBuilder errMsg = new StringBuilder();
-    errMsg.append(String.format("Unsupported Hive data type %s. ", unsupportedType));
+    errMsg.append("Unsupported Hive data type ").append(unsupportedType).append(". ");
```

Comment:
`System.getProperty("line.separator")` -> `System.lineSeparator()`
Also change this part of building string without reusing `errMsg` variable.